src/BenchmarkDotNet/Exporters/AsciiDocExporter.cs

```diff
@@ -17,11 +17,7 @@ private AsciiDocExporter()
         public override void ExportToLog(Summary summary, ILogger logger)
         {
             logger.WriteLine("....");
-            foreach (string infoLine in summary.HostEnvironmentInfo.ToFormattedString())
```

Comment:
This change will conflict with other PR: https://github.com/dotnet/BenchmarkDotNet/pull/893